edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Docker/CombinedDockerConfig.cs

```diff
@@ -9,11 +9,11 @@ namespace Microsoft.Azure.Devices.Edge.Agent.Docker
 
     public class CombinedDockerConfig
     {
-        public CombinedDockerConfig(string image, CreateContainerParameters createOptions, Option<NotaryContentTrust> notaryContentTrust, Option<AuthConfig> authConfig)
+        public CombinedDockerConfig(string image, CreateContainerParameters createOptions, Option<string> digest, Option<AuthConfig> authConfig)
         {
             this.Image = Preconditions.CheckNonWhiteSpace(image, nameof(image)).Trim();
             this.CreateOptions = Preconditions.CheckNotNull(createOptions, nameof(createOptions));
-            this.NotaryContentTrust = notaryContentTrust;
+            this.Digest = digest;
```

Comment:
What hash algorithm is to be used? Are we assuming SHA2?

Reply:
DockerConfig is the desired properties of the modules. It gets populated with whatever is in the deployment manifest. So what the customer sets the value of digest of the image to be. usually docker container images use SHS256

Reply:
Ok. We'll need to use the corresponding algorithm to calculate a digest against the downloaded payload; and then compare with this value. So the hash algorithm needs to be aligned. 

If we assume SHA2, then it should be the same. Or specify the algorithm in the config as well.

Reply:
The digest field would include the SHA2 algorithm along with it. For Example : The complete name would be **image-name@sha256:54f12e6b6ce**, then the digest that would be specified would be **sha256:54f12e6b6ce**
We wont calculate the digest in the device. This field will be used just to compare the digest received via the notary lookup command. So need to specify a separate field in config.